tensorflow/python/autograph/core/converter_testing.py

```diff
@@ -119,7 +119,7 @@ def transform(
         autograph_module=api)
 
     tr = TestingTranspiler(converter_module, ag_overrides)
-    transformed, _, _ = tr.transform_function(f, program_ctx)
```

Comment:
Nit: can we keep this unpacking? It's a bit more idiomatic.

Reply:
Strange, I don't see anything in the internal lint. At any rate, if the line remains unchanged, any error it might raise will be ignored.

Reply:
You need to check the log:


> Strange, I don't see anything in the internal lint. At any rate, if the line remains unchanged, any error it might raise will be ignored.

Can you check https://storage.cloud.google.com/tensorflow-nightly/prod/tensorflow/github/docker/code_check_changed_files_presubmit/1681/20220322-035621/index.html

Reply:
I see. That's a bug in the linter. Could you add a comment explaining this then?

Reply:
Done